code/modules/vending/wardrobes.dm

```diff
@@ -284,22 +294,21 @@
 	product_ads = "Any day above ground is a good one!;My day starts when yours ends!;And they call this a dying business!;See you when you're dead!"
 	vend_reply = "Don't forget your \"Buy one get one free\" burial deal!"
 	products = list(
-		/obj/item/storage/box/bodybags = 3,
-		/obj/item/storage/medkit/coroner = 1,
 		/obj/item/toy/crayon/white = 2,
-		/obj/item/clothing/mask/surgical = 1,
-		/obj/item/clothing/shoes/sneakers/black = 1,
```

Comment:
the sneakers are no long available, was this intentional?

Reply:
no it wasn't, I will fix that